test/powershell/Modules/Microsoft.PowerShell.Utility/XMLCommand.Tests.ps1

```diff
@@ -159,4 +159,97 @@ Describe "XmlCommand DRT basic functionality Tests" -Tags "CI" {
 		$cmd.Xml = $xml
 		$cmd.Xml | Should -Be $xml
 	}
+
+    It "ConvertFrom-CliXml directive should work" {
```

Comment:
What do you want test here?

Reply:
I copied the same set of tests that was present for ```Import-clixml``` and ```Export-clixml``` and made changes. I didn't write any test. This test seems to be like a basic functionality test.